support/rmt

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#############################################################
+# Name:        Supportconfig Plugin for RMT
+# Description: Gathers important troubleshooting information
+#              about RMT
+# License:     GPLv2
+# Author:      SCC Team <happy-customer@suse.de>
+# Modified:    2018 April 18
+#############################################################
+
+SVER=0.0.1
+RCFILE="/usr/lib/supportconfig/resources/scplugin.rc"
+
+[ -s $RCFILE ] && . $RCFILE || { echo "ERROR: Initializing resource file: $RCFILE"; exit 1; }
+
+## Our own helper functions
+validate_rpm_if_installed() {
+  THISRPM=$1
+  echo "#==[ Validating RPM ]=================================#"
+  if rpm -q "$THISRPM" >/dev/null 2>&1; then
+    echo "# rpm -V $THISRPM"
+
+    if rpm -V "$THISRPM"; then
+      echo "Status: Passed"
+    else
+      echo "Status: WARNING"
+    fi
+  else
+    echo "package $THISRPM is not installed"
+    echo "Status: Skipped"
+  fi
+  echo
+}
+
+RPMLIST="
+rmt-server
+nginx
+mariadb
+"
+SERVICES="rmt rmt-server-sync rmt-server-mirror mariadb nginx"
```

Comment:
Shouldn't this check for timers instead? `rmt-server-sync.timer` and `rmt-server-mirror.timer`. You could actually get statuses of both timers and services.

Reply:
Ah, I forgot we discussed that before standup.